pkg/asset/installconfig/clusterid.go

```diff
@@ -70,6 +71,7 @@ func generateInfraID(base string, maxLen int) string {
 
 	// truncate to maxBaseLen
 	if len(base) > maxBaseLen {
+		logrus.Warnf("Length of cluster name %s is %d which is greater than the max %d allowed. The name will be truncated to %s", base, len(base), maxBaseLen, strings.TrimRight(base[:maxBaseLen], "-"))
```

Comment:
What about:
```suggestion
		logrus.Warnf("Length of cluster name %q is %d which is greater than the max %d allowed. The name will be truncated to %q", base, len(base), maxBaseLen, strings.TrimRight(base[:maxBaseLen], "-"))
```
so the message is
```
WARNING Length of cluster name "rdossant-installer-05081037-very-long-name" is 42 which is greater than the max 21 allowed. The name will be truncated to "rdossant-installer-05"
```